### PR TITLE
[Workspace] Fix issues in workspace detail and association modal

### PR DESCRIPTION
**Pull request by Kapian1234**
### Description
Fix issues in workspace detail and association modal

## Screenshot
Screenshot 2024-09-18 at 14 50 05
Screenshot 2024-09-18 at 14 50 30
Screenshot 2024-09-18 at 14 51 02
Screenshot 2024-09-18 at 14 51 56

## Changelog

- fix: Fix issues in workspace detail and association modal

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by ruanyl** [comment]
@Kapian1234 Is this PR ready for review?

File: src/plugins/workspace/public/components/workspace_detail/association_data_source_modal.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+li[parentid] {
```
[the remark this answers is not in the document]
Comment (Kapian1234) [reply src/plugins/workspace/public/components/workspace_detail/association_data_source_modal.scss:1]: no, it just adds indentation to DQCs in association modal

[the remark this answers is not in the document]
Comment (ruanyl) [reply src/plugins/workspace/public/components/workspace_detail/association_data_source_modal.scss:1]: It is right? It applies to all `li` element with `parentid` attribute, but you only need to applies the style to the DQC  association modal if I understand correctly

**Review by ruanyl — CHANGES_REQUESTED** [review]
@wanglam Could you take a look at the comment and confirm?

File: src/plugins/workspace/public/components/workspace_detail/association_data_source_modal.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (wanglam) [reply src/plugins/workspace/public/components/workspace_detail/association_data_source_modal.scss:1]: Can we increase the div width in prepend element in [this line](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/workspace/public/components/workspace_detail/association_data_source_modal.tsx#L119)?  Then we don't need add this customized css.

[the remark this answers is not in the document]
Comment (Kapian1234) [reply src/plugins/workspace/public/components/workspace_detail/association_data_source_modal.scss:1]: Oh, I didn't consider that before. Add a div as prepend would be a better approach.

**Pull request closed without merging**